src/engine/sequencer.js

```diff
@@ -299,6 +299,28 @@ class Sequencer {
         }
     }
 
+    /**
+     * For sending the computational thread to the topblock, testing as of 5/6/21
+     */
+    stepToTopBlock (thread, branchNum, isLoop) {
+        // debugger
+        if (!branchNum) {
+            branchNum = 1;
+        }
+        const topBlockId = thread.topBlock;
+        // const branchId = thread.target.blocks.getBranch(
```

Comment:
Prefer not leave commented code in place.

Reply:
heard, taken care of